<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gerente</class>
 <widget class="QMainWindow" name="Gerente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>246</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="cadastrarButton">
          <property name="text">
           <string>Cadastrar funcionário</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="estoqueButton">
          <property name="text">
           <string>Estoque</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="financasButton">
          <property name="text">
           <string>Finanças</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="vendasButton">
          <property name="text">
           <string>Terminal de Vendas</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>246</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
